docs/library/theodore.md

```diff
@@ -4,66 +4,73 @@
 
 Theodore is a Thomson TO8D system emulator based on the Daniel Coulom's DCTO8D emulator. The Thomson TO8 is a home computer introduced by French company Thomson SA in 1986. The Thomson TO8D is an improved version that includes a built-in 3.5" floppy drive.
 
-### Author/License
-
 The Theodore core has been authored by
 
-- Thomas Lorblanchès
+- T. Lorblanchès
 
 The Theodore core is licensed under
 
 - [GPLv3](https://github.com/Zlika/theodore/blob/master/LICENSE)
 
 A summary of the licenses behind RetroArch and its cores have found [here](https://docs.libretro.com/tech/licenses/).
 
+## Requirements
+
+None
+
+## BIOS
+
+The Theodore core does not feature BIOS use.
+
 ## Extensions
 
 Content that can be loaded by the Theodore core have the following file extensions:
 
 - .fd (floppy disk)
+- .sap (floppy disk)
 - .k7 (tape)
 - .rom (cartridge)
+- .m7 (MEMO7 cartridge)
```

Comment:
*.m7 and *.rom extensions are used for the same thing, it's just that you can find on the net cartridge backups with these 2 extensions.